homeassistant/components/yamaha/media_player.py

```diff
@@ -245,7 +245,7 @@ def __init__(self, name, zctrl, source_ignore, source_names, zone_names):
                 self._attr_unique_id,
             )
         else:
-            _LOGGER.info(
+            _LOGGER.warning(
```

Comment:
We don't want logging in init methods at all. Please remove this logging or move it out of the init method. There should be no side effects when instantiating a class.